user_docs/en/userGuide.t2t

```diff
@@ -2537,7 +2537,7 @@ Thus, using the earlier example of replacing the word "bird" with "frog", if you
 
 A regular expression is a pattern containing special symbols that allow you to match on more than one character at a time, or match on just numbers, or just letters, as a few examples.
 Regular expressions are not covered in this user guide.
-For an introductory tutorial, please refer to [https://docs.python.org/3.7/howto/regex.html].
+For an introductory tutorial, please refer to [Python's Regular Expression HOWTO https://docs.python.org/3.7/howto/regex.html].

```

Comment:
Since this should go to master, might as well update the python version`

``suggestion
For an introductory tutorial, please refer to [Python's Regular Expression Guide https://docs.python.org/3.11/howto/regex.html].
```